human_detector:
  ros__parameters:
    debug_mode: true                      # Allows a window to popup to see the detection boxes
    alert_frequency: 5.0                  # In Hz (How many times in a second)
    save_face_frequency: 0.1              # Save/publish face In Hz, stops saving faces if set to 0.0
    camera_topic: "/image_raw"            # Camera topic name 
    saved_face_topic: ""                  # Broadcasts faces when detected, set to "" (empty string) to stop
    save_face_path: "/tmp/detected_faces" # Where to save the face pic (local path)
